openeuler/types.go

```diff
@@ -0,0 +1,84 @@
+package openeuler
+
+type Cvrf struct {
+	Title           string           `xml:"DocumentTitle"`
+	Tracking        DocumentTracking `xml:"DocumentTracking"`
+	Notes           []DocumentNote   `xml:"DocumentNotes>Note"`
+	ProductTree     ProductTree      `xml:"ProductTree"`
+	References      []Reference      `xml:"DocumentReferences>Reference"`
+	Vulnerabilities []Vulnerability  `xml:"Vulnerability"`
+}
+
+type DocumentTracking struct {
+	ID                 string     `xml:"Identification>ID"`
+	Status             string     `xml:"Status"`
+	Version            string     `xml:"Version"`
+	InitialReleaseDate string     `xml:"InitialReleaseDate"`
+	CurrentReleaseDate string     `xml:"CurrentReleaseDate"`
+	RevisionHistory    []Revision `xml:"RevisionHistory>Revision"`
+}
+
+type DocumentNote struct {
+	Text  string `xml:",chardata"`
+	Title string `xml:"Title,attr"`
+	Type  string `xml:"Type,attr"`
+}
+
+type ProductTree struct {
+	Branches []Branch `xml:"Branch"`
+}
+
+type Branch struct {
+	Type        string       `xml:"Type,attr"`
+	Name        string       `xml:"Name,attr"`
+	Productions []Production `xml:"FullProductName"`
+}
+
+type Production struct {
+	ProductID string `xml:"ProductID,attr"`
+	CPE       string `xml:"CPE,attr"`
+	Text      string `xml:",chardata"`
+}
+
+type Revision struct {
+	Number      string `xml:"Number"`
+	Date        string `xml:"Date"`
+	Description string `xml:"Description"`
+}
+
+type Vulnerability struct {
+	CVE             string   `xml:"CVE"`
+	Description     string   `xml:"Notes>Note"`
```

Comment:
It's better to use the name from the source
```suggestion
	Note     string   `xml:"Notes>Note"`
```